apps/mull-ui/src/app/components/navigation/top-nav-bar/top-nav-bar.scss

```
@@ -12,38 +13,31 @@
   flex-direction: row;
   align-items: center;
   justify-content: space-around;
-  padding: 0.7em 0;
+  padding: 0.5em 0;
   z-index: 3;
 }
-
-.logo {
+.top-nav-logo-container,
+.top-nav-logo {
+  align-self: center;
   display: block;
-  margin-left: auto;
-  margin-right: auto;
   width: 5.4rem;
 }
 
-.profile-button {
-  fill: $inactive-button-color;
-  position: absolute;
-  right: 1em;
-}
-
 .extra-buttons {
   display: none;
 }
 
 @media (min-width: $breakpoint-mobile) {
   .top-nav-container {
-    padding: 1em 0;
+    padding: 1rem 0;
```

I personally found that since the Müll logo is smaller, it looked a bit too "squished" towards the left. It looks a bit cleaner with more left-padding, but that's my opinion. Lmk what you think~
```suggestion
    padding: 2rem 0;
```

---

Oops I completely put this in the wrong place lol, I meant it here
```scss
.top-nav-logo-container {
    position: absolute;
    left: 2em;
    .top-nav-logo {
      width: 7em;
    }
  }
```

---

No worries I know where to put it. I'll have a look and see how it looks

---

I bumped it up to 1.6rem, I think 2 is a bit much, lemme know what you think when I finish my commit

---

Yep that's fine as well